CMakeLists.txt

```diff
@@ -145,15 +145,15 @@ endif()
 ## Specify additional locations of header files
 ## Your package locations should be listed before other locations
 # include_directories(include)
-include_directories(include 
+include_directories(include
```

Comment:
You have lots of weird whitespace issues, could you please fix these to minimize further agony if someone has to diff changes?